modules/lib/src/main/resources/assets/js/app/event/InspectEvent.ts

```diff
@@ -8,10 +8,14 @@ export class InspectEvent
 
     private readonly showPanel: boolean;
 
-    constructor(showWidget: boolean, showPanel: boolean) {
+    private readonly fromEditor: boolean;
```

Comment:
Change this to store PageNavigationEventSource instead

```suggestion
    private readonly source: PageNavigationEventSource;
```